lib/types.ts

```diff
@@ -14,3 +15,7 @@ export interface TokenResult {
   identity: string;
   accessToken: string;
 }
+
+export function isVideoCodec(codec: string): codec is VideoCodec {
+  return ['vp8', 'vp9', 'h264', 'av1'].includes(codec);
```

Comment:
haha, still don't know why you don't use the videoCodecs array from livekit-client, but I'll shut up

Reply:
copilot 💩